components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -221,7 +259,7 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
             kfctlLib: false,
             kfversion: Version.V06,
             // Version for local test. Staging and Prod with overwrite with their env vars.
-            kfversionList: [Version.V06, Version.V05],
+            kfversionList: ['v0.7.0', Version.V06],
```

Comment:
why not use the constant defined above for 0.7?

Reply:
The version here `v0.7.0` is for local test, version will be over written by env vars.
Above was a prefix `v0.7` to match v0.7.0 and sub-releases to new backend.